tools/tcat_ble_client/bbtc.py

```diff
@@ -49,9 +52,10 @@ async def main():
     logging.basicConfig(level=logging.WARNING)
 
     parser = argparse.ArgumentParser(description='Device parameters')
+    parser.add_argument('-a', '--adapter', help='Select HCI adapter')
     parser.add_argument('--debug', help='Enable debug logs', action='store_true')
     parser.add_argument('--info', help='Enable info logs', action='store_true')
-    parser.add_argument('--cert_path', help='Path to certificate chain and key', action='store', default='auth')
+    parser.add_argument('--cert-path', help='Path to certificate chain and key', action='store', default='auth')
```

Comment:
This should be reflected in expect tests as well. 
Please take a look into this https://github.com/openthread/openthread/blob/main/tests/scripts/expect/cli-tcat.exp (one of the reasons why CI is failing for you).